tests/unit_tests/test_tree.py

```diff
@@ -39,5 +43,105 @@ async def test_parsed_tree_empty_json():
         future.set_result(mocked_response)
         mocked_client.get.return_value = future
 
-        data = await parsed_tree(mocked_client, root_node_id)
+        data = await tree_from_vocabs(mocked_client, root_node_id)
         assert len(data) == 0
+
+
+def test_parse_tree():
+    node_id_biology = uuid.UUID("15fce411-54d9-467f-8f35-61ea374a298d")
+
+    query = tree_query(node_id_biology)
+    expected_query = {
+        "_source": [
+            "nodeRef.id",
+            "properties.cm:title",
+            "collections.path",
+            "parentRef.id",
+        ],
+        "from": 0,
+        "query": {
+            "bool": {
+                "filter": [{"term": {"path": node_id_biology}}],
+                "must": [
+                    {"match": {"permissions.Read": "GROUP_EVERYONE"}},
+                    {"match": {"properties.cm:edu_metadataset": "mds_oeh"}},
+                    {"match": {"nodeRef.storeRef.protocol": "workspace"}},
+                ],
+            }
+        },
+        "size": 500000,
+        "sort": ["fullpath"],
+    }
+
+    assert query.to_dict() == expected_query
+
+    if os.getenv("CI", False):
+        directory = "tests/unit_tests/resources"
```

Comment:
you can go with `directory = Path(__file__).parent.parent / "resources"` which should work in both scenarios as it identifies the location with an absolute path generated from the current file position. `Path` is part of python standard library. https://docs.python.org/3/library/pathlib.html

Reply:
I am uncertain about a) the double parent call, since this would lead us elsewhere, not to the child directories and b) that we can manage both cases in one.

Reply:
:D By reconfiguring my PyCharm test environment, I got it to work with the same directory for both, so I will - at least here - not yet use Path. But probably soon it will make sense to have some more ... clean way of path handling :)